Store api general updates (#2)
shopware/store-api-reference: docs/guides/faq.md

> }
> ```

> **How can I check whether I am currently logged in?**

Author: Here the auxiliary verb is missing in the sentence:  How can I check whether the user is currently logged in?

> **How can I check whether I am currently logged in?**

Reviewer: Do mean like this ?:
> How to check whether a user is currently logged in.

> **How can I check whether I am currently logged in?**

Author: Yes right and with a question mark please. :)

> **How can I check whether I am currently logged in?**

Reviewer: ```suggestion
**How to check whether a user is currently logged in?**
```

> > The least you need in order to have logged in is, a `sw-context-token` header. In order to check whether you are actually logged in call the following endpoint

Author: Suggesting to keep the sentence as follows: 
The least you need to get logged in is an `sw-context-token` header. In order to check whether the user is actually logged in, call the following endpoint

> > The least you need in order to have logged in is, a `sw-context-token` header. In order to check whether you are actually logged in call the following endpoint

Reviewer: Sounds good :+1:

**My customFields only contain IDs and not the objects?**

> Oftentimes, custom fields are used to persist data \(such as images or files\) along with products or categories. However, Shopware doesn't hydrate these as objects when products are fetched, because it cannot ensure that they are valid references or objects. Custom fields can contain any value and are only hydrated by the admin panel for management purposes. If you want to fetch the object \(e.g. a media object\) instead you can either [decorate the corresponding service](https://developer.shopware.com/docs/guides/plugins/plugins/plugin-fundamentals/adjusting-service) *(Developer Documentation)* or add these entities as an association using an [entity extension](https://developer.shopware.com/docs/guides/plugins/plugins/framework/data-handling/add-complex-data-to-existing-entities) *(Developer Documentation)*.

**How can I fetch SEO URLs for requested categories or products**

> > When you request include the `sw-include-seo-urls: 1` header. Shopware will automatically try to resolve relative URLs for all your categories and products. These will then be accessible in the `seoUrls` field of each record.  

Author: If your request includes the `sw-include-seo-urls: 1` header, Shopware will automatically try to resolve relative URLs for all your categories and products. These will then be accessible in the `seoUrls` field of each record.

> > When you request include the `sw-include-seo-urls: 1` header. Shopware will automatically try to resolve relative URLs for all your categories and products. These will then be accessible in the `seoUrls` field of each record.  

Reviewer: :heavy_check_mark:
